lnrpc/rpc.proto

```diff
@@ -244,3 +247,29 @@ message ShowRoutingTableRequest {
 message ShowRoutingTableResponse {
     repeated RoutingTableLink channels = 1;
 }
+
+message MultihopPaymentRequest {
```

Comment:
As mentioned above in the `cmd/lncli` section, I don't think this RPC is necessary.